src/features/base-layout/components/header/nav-item/nav-item.tsx

```diff
@@ -1,53 +1,53 @@
-import { useState } from 'react';
+import { ReactNode, useState } from 'react';
+import classNames from 'classnames/bind';
 import { NavLink } from 'react-router-dom';
-import { DropdownMenu } from '../dropdown/dropdown';
+import { DropdownWrapper } from '../dropdown/dropdown-wrapper';
 import { DropdownArrow } from '@/icons/dropdown-arrow';
 
 import styles from './nav-item.module.scss';
 
+const cx = classNames.bind(styles);
+
 type NavItemProps = {
   label: string;
-  href?: string;
-  dropdown?: boolean;
+  href: string;
+  dropdownInner?: ReactNode;
 };
 
-export const NavItem = ({ label, href, dropdown = false }: NavItemProps) => {
+export const NavItem = ({ label, href, dropdownInner }: NavItemProps) => {
   const [isDropdownOpen, setDropdownOpen] = useState(false);
 
   const onClose = () => setDropdownOpen(false);
   const onOpen = () => setDropdownOpen(true);
 
   return (
-    <>
-      {href && !dropdown ? (
-        <NavLink
-          className={({ isActive }) =>
-            isActive ? `${styles.menuItem} ${styles.active}` : styles.menuItem
-          }
-          to={href}
-          end>
-          <p className={styles.label}>{label}</p>
-        </NavLink>
-      ) : (
-        <>
-          <p
-            className={`${styles.menuItem} ${styles.dropdownToggle} ${isDropdownOpen ? styles.rotate : ''}`}
-            data-outside-click-ignore
-            onMouseLeave={onClose}
-            onMouseEnter={onOpen}>
-            <span className={styles.label}>{label}</span>
-            <span className={styles.dropdownArrow}>
-              <DropdownArrow />
-            </span>
-          </p>
-          <DropdownMenu
-            onMouseLeave={onClose}
-            onMouseEnter={onOpen}
-            isOpen={isDropdownOpen}
-            onClose={onClose}
-          />
-        </>
+    <div className={cx('menu-item-wrapper')}>
+      <NavLink
+        to={href}
+        className={({ isActive }) =>
+          cx(
+            'menu-item',
+            isActive ? 'active' : '',
+            dropdownInner ? 'dropdown-toggle' : '',
+            isDropdownOpen ? 'rotate' : '',
+          )
```

Comment:
```suggestion
          cx('menu-item', {
            active: isActive,
            'dropdown-toggle': dropdownInner,
            rotate: isDropdownOpen,
          })
```

Reply:
Corrected in [refactor: 256 - fix conditional className](https://github.com/rolling-scopes/site/pull/258/commits/261675056434d027d3fb5d96c69276b490b02f0a)